test/support/factory.ex

```diff
@@ -23,10 +23,20 @@ defmodule Wanda.Factory do
     %Catalog.Check{
       id: UUID.uuid4(),
       name: Faker.StarWars.character(),
+      group: Faker.Company.name(),
+      description: Faker.Lorem.sentence(),
+      remediation: Faker.Lorem.sentence(),
+      metadata: %{
+        target_type: Faker.StarWars.character(),
+        cluster_type: Faker.Lorem.sentence(),
+        provider:
+          Enum.take_random(["azure", "nutanix", "kvm", "vmware, gcp, aws"], Enum.random(0..6))
```

Comment:
I think that you don't need this `Enum.random(0..6)`.
`Enum.random(["azure", "nutanix", "kvm", "vmware, gcp, aws"])` should be enough

Reply:
It is not required, my idea was to populate a random amount of provider to simulate the structure of a check.

Example out of checks: 
`  provider: [azure, nutanix, kvm, vmware]`
  
So we can also remove Enum.random(0..6) and make a fix number. No strong opinion here :+1: